packages/web-application/src/web-application.ts

```diff
@@ -901,16 +648,20 @@ export class WebApplication extends PluggableModule {
             }
 
             try {
-                let element = getElementByXPath(xpath);
-                let evt = document.createEvent('HTMLEvents');
+                const element = getElementByXPath(xpath);
 
                 if (element) {
-                    element.focus();
                     element.value = value;
+                    const eventInit = {
+                        bubbles: true,
+                    };
+                    const inputEvent = new Event('input', eventInit);
+                    const keydownEvent = new Event('keydown', eventInit);
+                    const keyupEvent = new Event('keyup', eventInit);
 
-                    evt.initEvent('input', true, true);
-                    element.dispatchEvent(evt);
-                    element.blur();
+                    element.dispatchEvent(keydownEvent);
```

Comment:
After keydown there is a keypress event
https://developer.mozilla.org/ru/docs/Web/API/KeyboardEvent
It's not that critical but nice to have